testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CLITestUtil.java

```diff
@@ -76,7 +78,16 @@ public static CommandContext getCommandContext(String address, int port, InputSt
         // to avoid the need to reset the terminal manually after the tests, e.g. 'stty sane'
         System.setProperty("aesh.terminal","org.jboss.aesh.terminal.TestTerminal");
         setJBossCliConfig();
-        return CommandContextFactory.getInstance().newCommandContext(address + ":" + port, isRemote ? username : null, isRemote ? password.toCharArray() : null, in, out);
+        return CommandContextFactory.getInstance().newCommandContext(new CommandContextConfiguration.Builder()
+                .setController(address + ":" + port)
+                .setUsername(isRemote ? username : null)
+                .setPassword(isRemote ? password.toCharArray() : null)
+                .setConsoleInput(in)
+                .setConsoleOutput(out)
+                .setDisableLocalAuth(false)
+                .setInitConsole(false)
+                .setConnectionTimeout(TimeoutUtil.adjust(20 * 1000))
```

Comment:
There should not be any hardcoded timeouts in the testsuite. Every timeout should be adjusted by TimeoutUtil. That's also the reason why JBEAP-8244 is failing. You are, however, right that I don't need to increase the connectionTimeout for other tests, I will rework it a bit.